### PR TITLE
use analysis and collection in -tag

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

File: recipes-tag/analysis/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
 
 build:
   number: 0
 
 requirements:
   run:
     - album
-    - amostra  # [py3k]
-    - bluesky  # [py3k]
+    - amostra ==0.2
```
Comment (tacaswell) [inline recipes-tag/analysis/meta.yaml]: Why does analysis depend on bluesky?

Comment (tacaswell) [inline recipes-tag/analysis/meta.yaml]: and it should keep the `# [py3k]` selector.

File: recipes-tag/analysis/meta.yaml [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
-    - bluesky  # [py3k]
+    - amostra ==0.2  # [py3k]
+    - bluesky ==0.8.0  # [py3k]
     - channelarchiver
-    - databroker
-    - datamuxer
-    - ipywidgets
+    - databroker ==0.7.0
+    - datamuxer ==0.3.0
+    - doct ==1.0.3  # [py3k]
```
Comment (danielballan) [inline recipes-tag/analysis/meta.yaml]: Why is this py3k? It's used by databroker.

**Comment by licode** [comment]
Done!

**Pull request merged**